dash/_configs.py

```diff
@@ -4,47 +4,50 @@
 from . import exceptions
 from ._utils import AttributeDict
 
+load_dash_env_vars = lambda: AttributeDict(
+    {
+        var: os.getenv(var, os.getenv(var.lower()))
+        for var in (
+            'DASH_APP_NAME',
+            'DASH_URL_BASE_PATHNAME',
+            'DASH_ROUTES_PATHNAME_PREFIX',
+            'DASH_REQUESTS_PATHNAME_PREFIX',
+            'DASH_SUPPRESS_CALLBACK_EXCEPTIONS',
+            'DASH_ASSETS_EXTERNAL_PATH',
+            'DASH_INCLUDE_ASSETS_FILES',
+            'DASH_COMPONENTS_CACHE_MAX_AGE',
+            'DASH_INCLUDE_ASSETS_FILES',
+            'DASH_SERVE_DEV_BUNDLES',
+            'DASH_DEBUG',
+            'DASH_HOT_RELOAD',
+            'DASH_HOT_RELOAD_INTERVAL',
+            'DASH_HOT_RELOAD_WATCH_INTERVAL',
+            'DASH_HOT_RELOAD_MAX_RETRY',
+            'DASH_SILENCE_ROUTES_LOGGING',
+        )
+    }
+)
+
+DASH_ENV_VARS = load_dash_env_vars()
 
-def env_configs():
-    """
-    Configs from the environ.
-
-    :return: A dict with the dash environ vars
-    """
-    return AttributeDict({x: os.getenv(x, os.getenv(x.lower())) for x in (
-        'DASH_APP_NAME',
-        'DASH_URL_BASE_PATHNAME',
-        'DASH_ROUTES_PATHNAME_PREFIX',
-        'DASH_REQUESTS_PATHNAME_PREFIX',
-        'DASH_SUPPRESS_CALLBACK_EXCEPTIONS',
-        'DASH_ASSETS_EXTERNAL_PATH',
-        'DASH_INCLUDE_ASSETS_FILES',
-        'DASH_COMPONENTS_CACHE_MAX_AGE',
-        'DASH_INCLUDE_ASSETS_FILES',
-        'DASH_SERVE_DEV_BUNDLES',
-        'DASH_DEBUG',
-        'DASH_HOT_RELOAD',
-        'DASH_HOT_RELOAD_INTERVAL',
-        'DASH_HOT_RELOAD_WATCH_INTERVAL',
-        'DASH_HOT_RELOAD_MAX_RETRY',
-        'DASH_SILENCE_ROUTES_LOGGING'
-    )})
-
-
-def get_config(config_name, init, env, default=None, is_bool=False):
-    if init is not None:
-        return init
-
-    env_value = env.get('DASH_{}'.format(config_name.upper()))
-    if env_value is None:
+def get_combined_config(name, val, default=None):
```

Comment:
simplified the get_config by removing unnecessary parameters passing of env, and removed is_bool